File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.py
class Solution:
    def check(self, nums: List[int]) -> bool:
        n=len(nums)
        if n<=1:
            return True
        ic=0
        for i in range(1,n):
            if nums[i]<nums[i-1]:
                ic+=1
        if nums[0]<nums[n-1]:
            ic+=1
        return ic<=1

File: 1878-check-if-array-is-sorted-and-rotated/1878-check-if-array-is-sorted-and-rotated.cpp
class Solution {
public:
    bool check(vector<int>& nums) {
        int ic=0;
        for(int i=1;i<nums.size();i++){
            if(nums[i]<nums[i-1]){
                ic++;
            }
        }
        if(nums[0]<nums[nums.size()-1]) ic++;
        return ic<=1;
    }
};